### PR TITLE
Flush lines cleanup

### PR DESCRIPTION
**Pull request by lbudai**
* make global `flush_timeout` deprecated
* in http rename
  * `flush_bytes` to `batch_bytes`
  * `flush_lines` to `batch_lines`
  * `flush_timeout` to `batch_timeout`

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build FAILURE

File: lib/cfg-grammar.y [hunk 1]
```diff
@@ -1189,6 +1191,8 @@ threaded_dest_driver_option
         {
           log_threaded_dest_driver_set_max_retries(last_driver, $3);
         }
+        | KW_BATCH_LINES '(' nonnegative_integer ')' { log_threaded_dest_driver_set_flush_lines(last_driver, $3); }
```
Comment (furiel) [inline lib/cfg-grammar.y]: Can you rename the setters too? log_threaded_dest_driver_set_flush_lines -> ..._set_batch_lines

File: modules/http/http-grammar.ym [hunk 2]
```diff
@@ -119,9 +119,7 @@ http_option
     | KW_BODY       '(' template_content ')'  { http_dd_set_body(last_driver, $3); log_template_unref($3); }
     | KW_ACCEPT_REDIRECTS '(' yesno ')'       { http_dd_set_accept_redirects(last_driver, $3); }
     | KW_TIMEOUT '(' nonnegative_integer ')'  { http_dd_set_timeout(last_driver, $3); }
-    | KW_FLUSH_LINES '(' nonnegative_integer ')' { log_threaded_dest_driver_set_flush_lines(last_driver, $3); }
-    | KW_FLUSH_BYTES '(' nonnegative_integer ')' { http_dd_set_flush_bytes(last_driver, $3); }
-    | KW_FLUSH_TIMEOUT '(' nonnegative_integer ')' { log_threaded_dest_driver_set_flush_timeout(last_driver, $3); }
+    | KW_BATCH_BYTES '(' nonnegative_integer ')' { http_dd_set_flush_bytes(last_driver, $3); }
```
Comment (furiel) [inline modules/http/http-grammar.ym]: http_dd_set_flush_bytes > batch_bytes

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**